lib/aoc_rb/puzzle_source.rb

```diff
@@ -5,7 +5,7 @@ module AocRb
   module PuzzleSource
     extend self
 
-    def create_puzzle(year, day, input)
+    def create_puzzle(year, day, input = nil)
```

Comment:
Input should never be `nil` ... why would we want to provide this default?

Reply:
You're absolutely right, I was trying to figure out why the `output` method wasn't working and this was what I did to temporarily make the method not crash